drizzlepac/haputils/product.py

```diff
@@ -261,7 +261,7 @@ def align_to_gaia(
                     refname = (
                         self.product_basename + "_" + catalog_item + "_ref_cat.ecsv"
                     )
-
+                    self.refname = refname
```

Comment:
Trying to understand the code: the comment above says `Override the default self.refname` (and I understand this is not a new comment). But it looks like that's the first assignment to a new attribute `self.refname`. Am I misreading the code?
If not, I think the attribute should be defined in the `__init__` with some default value and the reassigned here,